src/DBAL/QueryFunction.php

```diff
@@ -0,0 +1,357 @@
+    }
+
+    /**
+     * Build a CONVERT SQL function call
+     * @param string|QueryExpression $expression Expression to convert
+     * @param string $transcoding Transcoding to use
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function convert(string|QueryExpression $expression, string $transcoding, ?string $alias = null): QueryExpression
+    {
+        global $DB;
+        $expression = $expression instanceof QueryExpression ? $expression : $DB::quoteName($expression);
+        return new QueryExpression('CONVERT(' . $expression . ' USING ' . $transcoding . ')', $alias);
+    }
+
+    /**
+     * Build a NOW SQL function call
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function now(?string $alias = null): QueryExpression
+    {
+        return new QueryExpression('NOW()', $alias);
+    }
+
+    /**
+     * Build a REPLACE SQL function call
+     * @param string|QueryExpression $expression Expression to search in
+     * @param string|QueryExpression $search String to search
+     * @param string|QueryExpression $replace String to replace
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function replace(string|QueryExpression $expression, string|QueryExpression $search, string|QueryExpression $replace, ?string $alias = null): QueryExpression
+    {
+        return self::getExpression('REPLACE', [$expression, $search, $replace], $alias);
+    }
+
+    /**
+     * Build a FROM_UNIXTIME SQL function call
+     * @param string|QueryExpression $expression Expression to convert
+     * @param string|QueryExpression|null $format Function result alias
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function fromUnixTimestamp(string|QueryExpression $expression, string|QueryExpression $format = null, ?string $alias = null): QueryExpression
+    {
+        $params = [$expression];
+        if ($format !== null) {
+            $params[] = $format;
+        }
+        return self::getExpression('FROM_UNIXTIME', $params, $alias);
+    }
+
+    /**
+     * Build a DATE_FORMAT SQL function call
+     * @param string|QueryExpression $expression Expression to format
+     * @param string|QueryExpression $format Format to use
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function dateFormat(string|QueryExpression $expression, string|QueryExpression $format, ?string $alias = null): QueryExpression
+    {
+        return self::getExpression('DATE_FORMAT', [$expression, $format], $alias);
+    }
```

Comment:
According to MySQL documentation (https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_date-format), `$format` will always be a string surrounded by `'`. We could automatize quoting of this parameter to make method usage simplier.

```suggestion
    public static function dateFormat(string|QueryExpression $expression, string $format, ?string $alias = null): QueryExpression
    {
        global $DB;
        $format = new QueryExpression($DB::quoteValue($format));
        return self::getExpression('DATE_FORMAT', [$expression, $format], $alias);
    }
```